tracer/src/Datadog.Trace/Ci/Agent/ICIAgentlessWriterSender.cs

```diff
@@ -0,0 +1,17 @@
+// <copyright file="ICIAgentlessWriterSender.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Threading.Tasks;
+using Datadog.Trace.Ci.Agent.Payloads;
+
+namespace Datadog.Trace.Ci.Agent
+{
+    internal interface ICIAgentlessWriterSender
+    {
+        Task<bool> Ping();
```

Comment:
Does the agentless writer need a ping operation? Can we remove this?

Reply:
sure.

Reply:
done in [9d33ce9](https://github.com/DataDog/dd-trace-dotnet/pull/2402/commits/9d33ce985ebec2a5fafc43ad5d8d375751aab442)